app/models/saved_claim/education_benefits/va_10203.rb

```diff
@@ -9,29 +11,37 @@ class Submit10203EVSSError < StandardError
 
   # rubocop:disable Metrics/CyclomaticComplexity
   def after_submit(user)
-    create_stem_automated_decision(user) if user.present?
+    @user = user
```

Comment:
Why not initialize user in an `initialize` method?

Reply:
User is passed in at https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/education_benefits_claims_controller.rb#L18

This was done here to minimize changes to existing creation and saving process when adding in the after submit process.

I switched from passing around `user` to `@user` to reduce method parameters needed for methods called inside the `after_submit` method

Adding user during initialize would happen in https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/models/saved_claim/education_benefits.rb and require additional testing beyond requested changes to make sure submission of other forms continues to work.